session-manager/src/session_manager/session_manager.py

```diff
@@ -85,12 +100,12 @@ def handle_user_message(
         Handles an incoming message, manages sessions and history, and returns a response.
         """
         if user_id not in self.user_sessions:
-            routing_agent_id = self.agents.get("routing-agent")
+            routing_agent_id = self.agents.get(self.ROUTING_AGENT_NAME)
```

Comment:
Maybe we can make this configurable through the agents configuration file at some point but that does not need to be in this PR.